retrofit_gen/lib/writer/writer.dart

```diff
@@ -54,6 +54,14 @@ class Writer {
       if (body is FormFieldBody) {
         sb.write('.urlEncodedFormField(${body.key}, ${body.name})');
       }
+
+      if (body is MultipartForm) {
+        sb.write('.multipart((serializers.to(${body.name}) as Map<String, dynamic>).map((key, value) => MapEntry(key, value.toString())))');
```

Comment:
` as Map<String, dynamic>).map((key, value) => MapEntry(key, value.toString())` is not needed. We can remove that.

Reply:
I'll try but without it doesn't work ^^ let me try